### PR TITLE
Storing peps, its related persons and links with companies

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/pep.py [hunk 1]
```diff
@@ -1,10 +1,385 @@
...
+                last_job_title_eng = pep_dict['last_job_title_en'].lower()
+            last_employer = None
+            if 'last_workplace' in pep_dict:
+                last_employer = pep_dict['last_workplace'].lower()
+            last_employer_eng = None
+            if 'last_workplace_en' in pep_dict:
+                last_employer_eng = pep_dict['last_workplace_en'].lower()
+            info = None
+            if 'wiki_uk' in pep_dict:
+                info = pep_dict['wiki_uk']
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:188]: this and all subsequent and similar blocks must be rewritten in a different format
```suggestion
            info = pep_dict.get('wiki_uk', None)
```

File: business_register/converter/pep.py [hunk 2]
```diff
@@ -1,10 +1,385 @@
...
+                    update_fields.append('last_job_title_eng')
+                if pep.last_employer != last_employer:
+                    pep.last_employer = last_employer
+                    update_fields.append('last_employer')
+                if pep.last_employer_eng != last_employer_eng:
+                    pep.last_employer_eng = last_employer_eng
+                    update_fields.append('last_employer_eng')
+                if pep.info != info:
+                    pep.info = info
+                    update_fields.append('info')
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:312]: too many repeated code, this should be automated

File: business_register/converter/pep.py [hunk 3]
```diff
@@ -1,10 +1,385 @@
...
+                assets_info = pep_dict['reputation_assets_uk']
+            assets_info_eng = None
+            if 'reputation_assets_en' in pep_dict:
+                assets_info_eng = pep_dict['reputation_assets_en']
+            criminal_proceedings = None
+            if 'reputation_crimes_uk' in pep_dict:
+                criminal_proceedings = pep_dict['reputation_crimes_uk']
+            criminal_proceedings_eng = None
+            if 'reputation_crimes_en' in pep_dict:
+                criminal_proceedings_eng = pep_dict['reputation_crimes_en']
```
Comment (roman-tiukh) [inline business_register/converter/pep.py:215]: too many repeated code, this should be automated

File: business_register/converter/pep.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/pep.py:188]: implemented, good idea!

**Pull request merged**